repo/blob/filesystem/filesystem_storage_capacity_windows.go

```diff
@@ -15,34 +15,23 @@ import (
 
 // nolint:gosec
 func (fs *fsStorage) GetCapacity(ctx context.Context) (blob.Capacity, error) {
-	dll, err := syscall.LoadLibrary("kernel32.dll")
-	defer syscall.FreeLibrary(dll) // nolint:errcheck
+	dll := syscall.MustLoadDLL("kernel32.dll")
+	getDiskFreeSpaceExW := dll.MustFindProc("GetDiskFreeSpaceExW")
```

Comment:
it can be even simpler, since there's already w windows wrapper that does GetDiskFreeSpaceExW

    var free, total, avail uint64

    path = "c:\\"
    pathPtr, err := windows.UTF16PtrFromString(path)
    if err != nil{
        panic(err)
    }
    err = windows.GetDiskFreeSpaceEx(pathPtr, &free, &total, &avail)